blocks/video-player-block/features/video-player/default.jsx

```diff
@@ -1,182 +1,142 @@
-	const customFieldSource = customFields?.itemContentConfig?.contentService ?? null;
-	const contentConfigValues = customFields?.itemContentConfig?.contentConfigValues;
-	// Support for deprecated 'websiteURL' custom field (use 'content-api' & URL for fetch)
-	const fetchSource = doFetch ? (!!websiteURL && "content-api") || customFieldSource : null;
-	const fetchDataQuery = websiteURL
-		? {
-				website_url: websiteURL,
-				site: arcSite,
-		  }
-		: contentConfigValues || null;
+	const { globalContent = {}, arcSite } = useFusionContext();
 
-	const fetchedData = useContent({
-		source: fetchSource,
-		query: fetchDataQuery,
-	});
-
-	embedHTML = doFetch ? fetchedData && fetchedData.embed_html : embedHTML;
+	const fetchQuery = websiteURL
+		? { website_url: websiteURL, site: arcSite }
+		: customFields?.itemContentConfig?.contentConfigValues || null;
 
-	useEffect(() => {
-		if (document.getElementById(`video-${videoRef.current}`)) {
-			const powaEl = document.getElementById(`video-${videoRef.current}`).firstElementChild;
+	// Support for deprecated 'websiteURL' custom field (use 'content-api' & URL for fetch)
+	const fetchSource = websiteURL
+		? "content-api"
+		: customFields?.itemContentConfig?.contentService || null;
 
-			if (powaEl) {
-				if (window.powaBoot) window.powaBoot();
-			}
-		}
+	const fetchedData = useContent({
+		query: inheritGlobalContent ? null : fetchQuery,
+		source: inheritGlobalContent ? null : fetchSource,
 	});
 
-	const { headlineBasic, descriptionBasic, credits } =
-		getFetchedPassedOrInheritedTitleDescriptionCaption(
-			globalContent,
-			inheritGlobalContent,
-			title,
-			description,
-			doFetch,
-			fetchedData
-		);
-
-	return (
-		<div className="container-fluid">
-			{alertBadge && (
-				<div className="padding-sm-bottom">
-					<AlertBadge>{alertBadge}</AlertBadge>
-				</div>
-			)}
-			{title && (
-				<PrimaryFont as="h2" className="xl-promo-headline">
-					{title}
-				</PrimaryFont>
-			)}
-			{embedHTML && (
-				<VideoPlayerPresentational
-					id={id}
-					embedMarkup={embedHTML}
-					enableAutoplay={enableAutoplay}
-					shrinkToFit={customFields?.shrinkToFit}
-					viewportPercentage={customFields?.viewportPercentage}
-					customFields={{
-						autoplay,
-						playthrough,
-					}}
-					displayTitle={!hideVideoTitle}
-					displayCaption={!hideVideoCaption}
-					displayCredits={!hideVideoCredits}
-					subtitle={headlineBasic}
-					caption={descriptionBasic}
-					credits={credits}
-				/>
-			)}
-			{description && (
-				<SecondaryFont as="p" className="description-text">
-					{description}
-				</SecondaryFont>
-			)}
-		</div>
-	);
-};
+	const contentSource = inheritGlobalContent ? globalContent : fetchedData;
+
+	const captionTitle = inheritGlobalContent
+		? contentSource?.headlines?.basic
+		: (description && title) || contentSource?.headlines?.basic;
+
+	const captionDescription = inheritGlobalContent
+		? contentSource?.descriptions?.basic
```

Comment:
```suggestion
		? contentSource?.description?.basic
```
![Screen Shot 2022-05-11 at 17 03 14](https://user-images.githubusercontent.com/5950956/167955065-8923f6e0-45c9-4aea-abe0-5abcb08a60b3.png)

the object that's coming in is description. that's why the content source is never showing the "These are tree blossoms" caption